### PR TITLE
Added ability to fill move fields by clicking.

### LINKED ISSUE #14: Add Front End Moving By Clicking

### PR DESCRIPTION
**Pull request by walshb421**
closes #14

### CODE REVIEW

File: src/components/Board.vue
Comment (GarettOS) [inline src/components/Board.vue:1]: nice

**Comment by GarettOS** [comment]
looks good

**Comment by walshb421** [comment]
> looks good

@GarettOS  Did you mean to approve this? We'll definitely revisit this when Bryonna makes more assets but I want to get working on the architecture things first

**Review by Bryonnab — APPROVED** [review]
👍

**Pull request merged**